test/test_document.rb

```diff
@@ -492,4 +492,33 @@ def assert_equal_value(key, one, other)
       assert_equal true, File.file?(@dest_file)
     end
   end
+
+  context "a document in a collection with dash-separated numeric file name" do
```

Comment:
we don't really _need_ these tests because most others in this spec file will fail if the regex is wrong, but i added them for consistency